commands/issue.go

```diff
@@ -48,80 +56,146 @@ var (
 
 func init() {
 	cmdCreateIssue.Flag.StringVarP(&flagIssueMessage, "message", "m", "", "MESSAGE")
-	cmdCreateIssue.Flag.StringVarP(&flagIssueFile, "file", "f", "", "FILE")
+	cmdCreateIssue.Flag.StringVarP(&flagIssueFile, "file", "F", "", "FILE")
 	cmdCreateIssue.Flag.VarP(&flagIssueLabels, "label", "l", "LABEL")
 
 	cmdIssue.Flag.StringVarP(&flagIssueAssignee, "assignee", "a", "", "ASSIGNEE")
+	cmdIssue.Flag.StringVarP(&flagIssueState, "state", "s", "", "ASSIGNEE")
 
 	cmdIssue.Use(cmdCreateIssue)
 	CmdRunner.Use(cmdIssue)
 }
 
-/*
-  $ hub issue
-*/
-func issue(cmd *Command, args *Args) {
-	runInLocalRepo(func(localRepo *github.GitHubRepo, project *github.Project, gh *github.Client) {
-		if args.Noop {
-			ui.Printf("Would request list of issues for %s\n", project)
-		} else {
-			issues, err := gh.Issues(project)
-			utils.Check(err)
-			for _, issue := range issues {
-				var url string
-				// use the pull request URL if we have one
-				if issue.PullRequest.HTMLURL != "" {
-					url = issue.PullRequest.HTMLURL
-				} else {
-					url = issue.HTMLURL
-				}
+func listIssues(cmd *Command, args *Args) {
```

Comment:
This is a quite long function, consider splitting something out in another method. This would help a lot for documentation and understanding it